v1/api/swagger/paths/services@{serviceId}.yaml

```
@@ -34,6 +38,8 @@ put:
   description: >-
     Updates an OS service image with a matching ID.
     Note that Hoster or BMaaS Access Owner role is required for this operation.
```

This is GLCS only restriction. Still TODO on GLP side, but no need to comment on that.
```suggestion
    Note that Hoster or BMaaS Access Owner role is required for this operation under GreenLake Cloud Service.
```